test/Motor.Extensions.Hosting_IntegrationTest/GenericHostingTests.cs

```diff
@@ -67,8 +68,11 @@ public async Task
             var channel = _fixture.Connection.CreateModel();
             await CreateQueueForServicePublisherWithPublisherBindingFromConfig(channel);
 
+            var motorCloudEvent = MotorCloudEvent.CreateTestCloudEvent(Array.Empty<byte>());
+            motorCloudEvent.SetRabbitMQPriority(123);
```

Comment:
The priority isn't needed / tested any where, right? Did you use this for debugging or should this be tested somehow?

Reply:
I used this just to ensure that messages are still published if they have any extension set, regardless of its content. But now that there are more tests which include extensions, I could also drop this one.